doc/cascadia/profiles.schema.json

```diff
@@ -278,7 +280,7 @@
             "direction": {
               "$ref": "#/definitions/Direction",
               "default": "left",
-              "description": "The direction to move the pane separator in"
+              "description": "The direction to move the pane separator in. Direction cannot be 'next' or 'previous'."
```

Comment:
But `next` and `previous` are still displayed by the schema. I'm worried we'll get a few bug reports on that. So I recommend we definitely split the schema into enums.